torch/csrc/inductor/aoti_torch/opaque_tensor.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <ATen/Tensor.h>
+
+namespace torch {
+namespace aot_inductor {
+
+at::Tensor mkldnn_tensor_from_data_ptr(
```

Comment:
If `mkldnn_tensor_from_data_ptr` is only used by `aoti_torch_create_tensor_from_blob`, it's better to keep it as a utility function in shim_common.cpp. Files in `atoi_torch` are a set of limited interfaces used by the generated `model.so`, so adding a generic torch function here can cause confusion and people may use it by mistake.